Objects/regions.c

```diff
@@ -1282,3 +1278,61 @@ PyTypeObject PyRegion_Type = {
     0,                                       /* tp_alloc */
     PyType_GenericNew,                       /* tp_new */
 };
+
+static const char *get_region_name(PyObject* obj) {
+    if (_Py_IsLocal(obj)) {
+        return "Default";
+    } else if (_Py_IsImmutable(obj)) {
+        return "Immutable";
+    } else {
+        const regionmetadata *md = Py_REGION_DATA(obj);
+        return md->bridge->name
```

Comment:
I would prefer for the name to be on the regionmetadata object. The bridge pointer is a weak reference, so I am concerned if this could fail.

Reply:
Good suggestion, this also makes a lot of sense with the changes of region merging

Reply:
DONE